marimo/_plugins/ui/_impl/table.py

```diff
@@ -132,6 +132,17 @@ class GetDataUrlResponse:
     format: Literal["csv", "json", "arrow"]
 
 
+@dataclass
+class CalculateTopKRowsArgs:
+    column: ColumnName
+    k: int
+
+
+@dataclass
+class CalculateTopKRowsResponse:
+    data: Union[JSONType, str]
```

Comment:
i think we should just return the json `{"key": count}` or tuple: `[(key, number)]`. 

tuple might handle None/null better since keys get converted to strings